src/test/resources/cucumber/features/integrationTests/ast/astScanProcessing.feature

```diff
@@ -1,35 +1,44 @@
 @AstIntegrationTests @IntegrationTest
 Feature: Cx-Flow AST Integration permutation tests
-  
 
-  @ASTRemoteRepoScan 
-  Scenario Outline: scan multiple initiators
-    Given scan initiator list is "<initiators>"
-    Then the returned contain populated results for all initiators
+
+  @ASTRemoteRepoScan
+  Scenario Outline: using multiple vulnerability scanners
+    Given enabled vulnerability scanners are "<scanners>"
+    Then scan results contain populated results for all scanners
```

Comment:
when running scan

Given is for setup
When is for performing an action
Then is for validating results